src/scss/theme/_book.scss

```
@@ -59,14 +65,22 @@
         }
     }
 
-    &:hover .book-cover {
-        transform: rotateX(75deg) translateZ(3px) scale(0.75);
+    @media screen and (max-width: 767px) {
```

Lets use the full power of SCSS and Bootstrap:
https://getbootstrap.com/docs/5.3/layout/breakpoints/#max-width
According to this it should be:

```suggestion
    @include media-breakpoint-down(md) {
```

Also they are in the same level. Why there are 3 media queries instead of one?

---

done